extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -105,14 +105,19 @@ public function loadData() {
 			'data' 	=> array()
 		);
 
+		$context = RequestContext::getMain();
+		$user = $context->getUser();
+		$lang = $context->getLanguage();
+
 		$orderby = implode(",", $this->mOrder);
 		$subMemkey = array(
 			'G'  . implode(",", is_array($this->mFilterGroup) ? $this->mFilterGroup : array()),
 			'U'  . $this->mUserId,
 			'E'  . $this->mEditsThreshold,
 			'O'  . $this->mOffset,
 			'L'  . $this->mLimit,
-			'O'  . $orderby
+			'O'  . $orderby,
+			'L'  . $lang->getCode(), // localized messages are cached, vary by user language
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app